src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -72,24 +72,24 @@ enum OpcodeClient : uint32
     CMSG_ATTACKSTOP                                   = 0x065B,
     CMSG_ATTACKSWING                                  = 0x1E9B,
     CMSG_AUCTION_HELLO                                = 0x13EA,
-    CMSG_AUCTION_LIST_BIDDER_ITEMS                    = 0xBADD,
-    CMSG_AUCTION_LIST_ITEMS                           = 0xBADD,
-    CMSG_AUCTION_LIST_OWNER_ITEMS                     = 0xBADD,
-    CMSG_AUCTION_LIST_PENDING_SALES                   = 0xBADD,
-    CMSG_AUCTION_PLACE_BID                            = 0xBADD,
-    CMSG_AUCTION_REMOVE_ITEM                          = 0xBADD,
-    CMSG_AUCTION_SELL_ITEM                            = 0xBADD,
+    CMSG_AUCTION_LIST_BIDDER_ITEMS                    = 0x050E,
+    CMSG_AUCTION_LIST_ITEMS                           = 0x0B53,
+    CMSG_AUCTION_LIST_OWNER_ITEMS                     = 0x03E2,
+    CMSG_AUCTION_LIST_PENDING_SALES                   = 0x11E2,
+    CMSG_AUCTION_PLACE_BID                            = 0x0B11,
+    CMSG_AUCTION_REMOVE_ITEM                          = 0x1BEB,
+    CMSG_AUCTION_SELL_ITEM                            = 0x0027,
```

Comment:
And it's not a problem, SMSG opcodes can have the same values as CMSG with completely different function